concepts/bot_connector/channel.md

```diff
@@ -17,54 +17,54 @@ If a channel doesn't natively support a rich format, Bot Connector will handle i
                                                         style="font-size: 30px; width: 40px; margin: 0px 0.5rem;"></i>
         </th>
         <th width="50px" style="text-align: center;"><img class="custom mx-auto mb1 noShadow"
-                                                          src="https://www.recast.ai/assets/images/integrations/callr.png" width="30px"
+                                                          src="https://www.cai.tools.sap/assets/images/integrations/callr.png" width="30px"

```

Comment:
It seems all images are now stored in the CDN, can you change this kind of links to target the CDN?

Reply:
All /integrations/... images don't seem to be on the CDN yet. 
If someone with access could upload them that would be amazing. Then I will change the urls. 

For consistency it would be best to put them into /man/channels/

Reply:
it seems to be good !